test/verify/check-sosreport

```diff
@@ -43,6 +45,11 @@ only-plugins=release,date,host,cgroups,networking
 threads=1
 """)
 
+        # Sos 4 is very slow to start without Internet, but killing
+        # the repos fixes that.
+        self.restore_dir("/etc/yum.repos.d", reboot_safe=True)
```

Comment:
OOI, we don't reboot in this test, so it seems it wouldn't have to use `reboot_safe` (the default `reboot_safe=False` is much more robust)? Or does sosreport somehow stumble over the bind mount?

Reply:
Hrrm, that was just copy/paste...